lib/importers/protected_food_drink_name/parser.rb

```diff
@@ -13,7 +13,7 @@ def get_attributes
           class_category: class_category,
           protection_type: protection_type,
           # reason_for_protection: reason_for_protection, #TODO
```

Comment:
I think we should be able to uncomment this now, as the field is there. It was left commented out until we'd done this work!